yucca/functional/utils/torch_utils.py

```diff
@@ -14,7 +14,7 @@ def get_available_device():
     if torch.cuda.is_available():
         return "cuda"
     elif torch.backends.mps.is_available():
-        return "mps"
+        return "cpu"  # we cannot use mps, since Conv3D is not supported here.
```

Comment:
we can, just not for 3D, so this should be reverted

Reply:
But tests are in 3D, and this means tests doesn't run locally.

Reply:
Hmm, fair point and I guess this doesn't mess with manually setting training to MPS since we removed the device calls in training - and who would actually like to do things on MPS anyway tbh